src/frontend/components/Chat/StudentChat/index.tsx

```diff
@@ -1,21 +1,75 @@
 import { Box } from 'grommet';
-import React from 'react';
+import React, { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
 
 import { ChatConversationDisplayer } from 'components/Chat/SharedChatComponents/ChatConversationDisplayer';
-import { ChatInputBar } from 'components/Chat/SharedChatComponents/ChatInputBar';
+import { InputBar } from 'components/Chat/SharedChatComponents/InputBar';
+import { InputDisplayNameOverlay } from 'components/Chat/SharedChatComponents/InputDisplayNameOverlay';
+import { JoinChatButton } from 'components/Chat/SharedChatComponents/JoinChatButton';
+import { converse } from 'utils/window';
 
 export const StudentChat = () => {
+  const [overlay, setOverlay] = useState(false);
+  const [isInputBarActive, setInputBarActive] = useState(false);
+  const intl = useIntl();
+
+  const processChatMessage = (chatMsg: string) => {
+    converse.sendMessage(chatMsg);
+    return true;
+  };
+
+  const messages = defineMessages({
+    inputBarPlaceholder: {
+      defaultMessage: 'Message...',
+      description:
+        'The input bar to write, edit and send messages to the chat conversation.',
+      id: 'components.InputBar.inputBarPlaceholder',
+    },
+  });
+
+  const handleJoinChatButton = () => {
+    setOverlay(true);
+  };
+
   return (
     <Box
-      direction="column"
       border={{
         color: 'blue',
         size: 'small',
       }}
+      direction="column"
       height="medium"
     >
-      <ChatConversationDisplayer />
-      <ChatInputBar />
+      <Box
+        style={{
```

Comment:
you should use styled-component

Reply:
no needed anymore because I drop off use of display property and mount / unmount component instead.